odr-audioenc.spec

```diff
@@ -105,7 +105,7 @@ in form of a library to be used with the encoder for the ODR-mmbTools
 
 %package -n     %{toolame_dab_name}-devel
 Version:        %{libtoolame_dab_version}
-Release:        %{libtoolame_dab_release}.1
+Release:        %{libtoolame_dab_release}.2
```

Comment:
And another one that could be added to `%define libtoolame_dab_release`. This change also contradicts the release version of  `%{toolame_dab_name}-devel` vs. `%{toolame_dab_name}`.

Reply:
Properly incremented and fixed.